draft-ietf-quic-qlog-main-schema.md

```diff
@@ -492,6 +493,7 @@ JSON-SEQ serialization example:
     "serialization_format": "application/qlog+json-seq",
     "title": "Name of JSON Text Sequence qlog file (short)",
     "description": "Description for this trace file (long)",
+    "event_schemas": ["urn:ietf:params:qlog:events:quic"],
```

Comment:
Since the `protocol_types` field below is for both QUIC and H3, it would make sense for this to also include the HTTP/3 schema URI:

```suggestion
    "event_schemas": ["urn:ietf:params:qlog:events:quic", "urn:ietf:params:qlog:events:http3"],
```